rules/mockups/rule.md

```diff
@@ -0,0 +1,100 @@
+---
+type: rule
+title: Do you determine specifications with mockups?
+uri: mockups
+authors:
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
+  - title: Jayden Alchin
+    url: https://www.ssw.com.au/people/jayden-alchin
+  - title: Geordie Robinson
+    url: https://ssw.com.au/people/geordie-robinson
+related:
+  - storyboards
+created: 2023-09-04T07:37:31.732Z
+guid: 2b8c7067-4d59-49b2-a448-4b746b11383a
+---
+Many user requirements can be best encapsulated in visual mockups. A design mockup is  usually developed at the specification review phase to illustrates the layout, structure, and content of a product or interface before it is developed. A mockup can become a prototype over time but they tend to come earlier in the design process and are static images rather than interactive testing tools.
+
+<!--endintro-->
+
+There are 3 levels of mockups:
+
+1. **Low-fidelity Mockups** 
+2. **High-fidelity Mockups** (recommended)
```

Comment:
I don't think we can recommend one or another... it's based on the project requirements